app/controllers/concerns/sort_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SortHelper
```

Comment:
Can you update this to implement a sort_by method

Then also implement the sort_by_reverse, but it's implementation should be something like:

```ruby
def self.sort_by_reverse(collection: nil, attributes: nil, prepend_rejected: false)
  self.sort_by(
    collection: collection,
    attributes: attributes,
    prepend_rejected: prepend_rejected
  ).reverse
end
```

Reply:
🎉